spec/lib/solidus_paypal_commerce_platform/configuration_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
-RSpec.describe SolidusPaypalCommercePlatform::Environment do
-  let(:subject) { Class.new.extend described_class }
+RSpec.describe SolidusPaypalCommercePlatform::Configuration do
+  let(:subject) { SolidusPaypalCommercePlatform.config }
```

Comment:
Here, for better isolation and to avoid flakey specs due to execution order, would be better to have the subject defined as `described_class.new`. So each example execution will it's own configuration instance and the main one is not affected by chages.

Reply:
Ah, yeah, that's better. Thanks!